jwplayer-appletv-web-app/js/utils/Promise.js

```diff
@@ -0,0 +1,235 @@
+/*! https://github.com/taylorhakes/promise-polyfill v6.0.2 by Taylor Hakes | MIT license */
```

Comment:
See [the Promise polyfill license](https://github.com/taylorhakes/promise-polyfill/blob/master/LICENSE)